src/utils/narrow.js

```diff
@@ -124,3 +124,7 @@ export const narrowFromMessage = (message) => {
 
   return streamNarrow(message.display_recipient);
 };
+
+export const isNarrowWithComposeBox = (narrow) => {
```

Comment:
We already have such a call (and yours have a bug ;) ).
Existing one is `canSendToNarrow`.